typescript/vitest.config.js

```diff
@@ -4,5 +4,6 @@ export default defineConfig({
   test: {
     testTimeout: 300000,
     hookTimeout: 300000,
```

Comment:
do you need these long timeouts on unit tests?

Reply:
Probably not. They could be reduced a bit.